src/main/java/com/getcloudcherry/survey/helper/RecordAnswer.java

```diff
@@ -126,14 +147,16 @@ public SurveyAnswers getAnswers() {
     public boolean checkIfPreFillExists(SurveyQuestions iQuestion) {
 
         for (String aTag : iQuestion.questionTags) {
```

Comment:
For 'Scale' type question(NPS) by default, we have to show legends.
<img width="461" alt="npslegends" src="https://cloud.githubusercontent.com/assets/7541895/17745571/ec904248-64c9-11e6-93f6-0aee07ad95d4.PNG">

We should also support custom legends, one legend starts below 0 and other ends below 10.

**Sample JSON for Custom Legends in NPS:**
"multiSelect":[
"0;Not Likely-100;Extremely Likely"
]